meta/.phpstorm.meta.php

```diff
@@ -136,6 +136,12 @@ function argumentsSet($setName) {
 	expectedArguments(\fopen(), 1, 'r', 'r+', 'w', 'w+', 'a', 'a+', 'x', 'x+', 'c', 'c+', 'e');
 	expectedArguments(\popen(), 1, 'r', 'r+', 'w', 'w+', 'a', 'a+', 'x', 'x+', 'c', 'c+', 'e');
 	expectedArguments(\SplFileInfo::openFile(), 0, 'r', 'r+', 'w', 'w+', 'a', 'a+', 'x', 'x+', 'c', 'c+', 'e');
+    
+    registerArgumentsSet('socket_transports',
+        'tcp://', 'udp://', 'ssl://', 'sslv2://', 'sslv3://', 'tls://', 'unix://', 'udg://'
+    );
+    expectedArguments(\fsockopen(), 0, argumentsSet('socket_transports'));
```

Comment:
So this one is actually misusage  - we want to provide actual expected arguments rather then help to complete part of the some of possible values.

Reply:
I understand.
But why not allow something like this to help with prefixes and similar?

Reply:
We want to extend the feature in the future and this concept contradicts it's purpose. It should be implemented separately by another instruments.